app/models/sections/info/DestinationType.scala

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.sections.info
+
+import models.{Enumerable, WithName}
+import play.api.i18n.Messages
+import uk.gov.hmrc.govukfrontend.views.viewmodels.content.Text
+import uk.gov.hmrc.govukfrontend.views.viewmodels.radios.RadioItem
+
+sealed trait DestinationType
+
+object DestinationType extends Enumerable.Implicits {
+
+  case object TemporaryRegisteredConsignee extends WithName("1") with  DestinationType
```

Comment:
What are the other valid DestinationTypes? As it would be good to add them all in here now so that this model is complete.

Reply:
These are the valid options and values from the XSD for the `tcl` types:
https://github.com/hmrc/emcs/blob/main/emcs-xmlbeans/src/main/resources/schemas/tcl.xsd#L1073-L1137

I think this might have already been created as a model in the backend too:
https://github.com/hmrc/emcs-tfe/blob/main/app/uk/gov/hmrc/emcstfe/models/common/DestinationType.scala

So I would just steal that and tweak where needed maybe.